### PR TITLE
#2880 Miscellaneous corrections

### PR DESCRIPTION
**Pull request by ErwanGuichard49**
closes [#2880](https://github.com/dynawo/dynawo/issues/2880)

### CODE REVIEW

**Review by joyelfeghali — CHANGES_REQUESTED** [review]

File: dynawo/sources/Models/Modelica/Dynawo/Electrical/Controls/IEC/WPP/WPPControl.mo [hunk 1]
```diff
@@ -134,11 +130,11 @@ model WPPControl "Control model for IEC N°61400-27-1 standard WPP"
     Placement(visible = true, transformation(origin = {40, -60}, extent = {{-20, -20}, {20, 20}}, rotation = 0)));
   Modelica.Blocks.Sources.Constant const(k = 0) annotation(
     Placement(visible = true, transformation(origin = {-10, 100}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
-  Dynawo.Electrical.Controls.IEC.BaseControls.WPP.LinearCommunication linearCommunicationWPRef(X0Pu = {-P0Pu * (SystemBase.SnRef / SNom), X0Pu}, nu = 2, tLag = tLag, tLead = tLead)  annotation(
+  Dynawo.Electrical.Controls.IEC.BaseControls.WPP.LinearCommunication linearCommunicationWPRef(X0Pu = {-P0Pu * (SystemBase.SnRef / SNom), XWT0Pu}, nu = 2, tLag = tLag, tLead = tLead) annotation(
```
[the remark this answers is not in the document]
Comment (ErwanGuichard49) [reply dynawo/sources/Models/Modelica/Dynawo/Electrical/Controls/IEC/WPP/WPPControl.mo:133]: This was a rebasing error whereby I had kept the original formulation. It has now been reverted to X0Pu.

File: dynawo/sources/Models/Modelica/Dynawo/NonElectrical/Blocks/Continuous/VarLimPIDFreeze.mo [hunk 2]
```diff
@@ -58,7 +57,7 @@ model VarLimPIDFreeze "PI controller with limited output (with adjustable limits
     Placement(transformation(extent = {{-10, -10}, {10, 10}})));
   Blocks.Math.Add3 addI(k2 = -1) annotation(
```
[the remark this answers is not in the document]
Comment (ErwanGuichard49) [reply dynawo/sources/Models/Modelica/Dynawo/NonElectrical/Blocks/Continuous/VarLimPIDFreeze.mo:58]: I have removed the import lines related to Blocks and kept the others, as they shorten the equations.

**Review by joyelfeghali — CHANGES_REQUESTED** [review]
nrt is OK?

**Comment by ErwanGuichard49** [comment]
> nrt is OK?

Yes save for _examples/DynaSwing/IEC/Wind/Neplan/WPP4ACurrentSourceU/WPP4ACurrentSource.jobs_ where the differences were invisible on the graph. It has been updated.

**Pull request merged**